.github/workflows/ci.yml

```
@@ -29,12 +29,13 @@ jobs:
         run: vcpkg/vcpkg install @vcpkg_x86-windows.txt
 
       - name: Build application
-        uses: ashutoshvarma/action-cmake-build@master
-        with:
-          build-dir: ${{ runner.workspace }}/openspades.mk
-          build-type: RelWithDebInfo
-          configure-options: -A Win32 -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake -D VCPKG_TARGET_TRIPLET=x86-windows-static
-          parallel: 8
+        run: |
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

```suggestion
        shell: bash
        run: |
```